mne/datasets/config.py

```diff
@@ -95,123 +95,273 @@
+    folder_name='MNE-epilepsy-ecog-data',
+    config_key='MNE_DATASETS_EPILEPSY_ECOG_PATH',
+)
+
+# Fieldtrip CMC dataset
+fieldtrip_cmc = dict(
+    archive_name='SubjectCMC.zip',
+    hash='md5:6f9fd6520f9a66e20994423808d2528c',
+    url='https://osf.io/j9b6s/download?version=1',
+    folder_name='MNE-fieldtrip_cmc-data',
+    config_key='MNE_DATASETS_FIELDTRIP_CMC_PATH'
+)
+
+# brainstorm datasets:
+bst_auditory = dict(
+    archive_name='bst_auditory.tar.gz',
+    hash='md5:fa371a889a5688258896bfa29dd1700b',
+    url='https://osf.io/5t9n8/download?version=1',
+    folder_name='MNE-brainstorm-data',
+    config_key='MNE_DATASETS_BRAINSTORM_PATH',
+)
+bst_phantom_ctf = dict(
+    archive_name='bst_phantom_ctf.tar.gz',
+    hash='md5:80819cb7f5b92d1a5289db3fb6acb33c',
+    url='https://osf.io/sxr8y/download?version=1',
+    folder_name='MNE-brainstorm-data',
+    config_key='MNE_DATASETS_BRAINSTORM_PATH',
+)
+bst_phantom_elekta = dict(
+    archive_name='bst_phantom_elekta.tar.gz',
+    hash='md5:1badccbe17998d18cc373526e86a7aaf',
+    url='https://osf.io/dpcku/download?version=1',
+    folder_name='MNE-brainstorm-data',
+    config_key='MNE_DATASETS_BRAINSTORM_PATH',
+)
+bst_raw = dict(
+    archive_name='bst_raw.tar.gz',
+    hash='md5:fa2efaaec3f3d462b319bc24898f440c',
+    url='https://osf.io/9675n/download?version=2',
+    folder_name='MNE-brainstorm-data',
+    config_key='MNE_DATASETS_BRAINSTORM_PATH',
+)
+bst_resting = dict(
+    archive_name='bst_resting.tar.gz',
+    hash='md5:70fc7bf9c3b97c4f2eab6260ee4a0430',
+    url='https://osf.io/m7bd3/download?version=3',
+    folder_name='MNE-brainstorm-data',
+    config_key='MNE_DATASETS_BRAINSTORM_PATH',
+)
+
+# HF-SEF
+hf_sef_raw = dict(
+    archive_name='hf_sef_raw.tar.gz',
+    hash='md5:33934351e558542bafa9b262ac071168',
+    url='https://zenodo.org/record/889296/files/hf_sef_raw.tar.gz',
+    folder_name='HF_SEF',
+    config_key='MNE_DATASETS_HF_SEF_PATH',
+)
+hf_sef_evoked = dict(
+    archive_name='hf_sef_evoked.tar.gz',
+    hash='md5:13d34cb5db584e00868677d8fb0aab2b',
+    url=('https://zenodo.org/record/3523071/files/'
+         'hf_sef_evoked.tar.gz'),
+    folder_name='HF_SEF',
+    config_key='MNE_DATASETS_HF_SEF_PATH',
+)
+
+# "fake" dataset (for testing)
+fake = dict(
+    archive_name='foo.tgz',
+    hash='md5:3194e9f7b46039bb050a74f3e1ae9908',
+    url=('https://github.com/mne-tools/mne-testing-data/raw/master/'
+         'datasets/foo.tgz'),
+    folder_name='foo',
+    config_key='MNE_DATASETS_FAKE_PATH'
+)
+
+# FreeSurfer files
+lh_aparc_sub_annot = dict(
+    archive_name='lh.aparc_sub.annot',
+    hash='md5:9e4d8d6b90242b7e4b0145353436ef77',
+)
+rh_aparc_sub_annot = dict(
+    archive_name='rh.aparc_sub.annot',
+    hash='md5:dd6464db8e7762d969fc1d8087cd211b'
+)
+
+# below, do **not** include the release
+# number in the names of the testing or
+# misc datasets (it gets done automatically
+# in mne/datasets/utils.py). Only update
+# the ' : 'md5 hashes as needed.
```

Comment:
```suggestion
```
seems like this warning is no longer necessary; it should be obvious not to change the variable name, and it is clear from the URL that the version will be `.format()`ted in later.